test/ambuda/views/proofing/test_talk.py

```diff
@@ -44,12 +44,12 @@ def test_create_post(rama_client):
     assert "Create post" in resp.text
 
 
-def test_create_post__bad_project(rama_client):
+def test_create_post__good_project(rama_client):
```

Comment:
the original names are incorrect here -- perhaps `__valid_thread` and `__missing_thread` instead.